### DISCUSSION TITLE
What is the best model that support Russian language?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [oobabooga/text-generation-webui#3431] [General]
What is the best model that support Russian language?

**Comment by flflow** [comment]
https://github.com/janvarev/multi_translate

**Comment by kreolsky** [comment]
> What is the best model that support Russian language?
I'm not sure, but maybe this: https://huggingface.co/fffrrt/ruGPT-3.5-13B-GPTQ

**Comment by DiDHack** [comment]
Command R+ на данный момент лучшая из доступных. По креативности она на втором месте
